wb-mqtt-serial-templates/config-wb-mrgbw-d-fw3.json

```diff
@@ -7363,6 +7375,11 @@
                 "dp_time_title": "Double Press Time (ms)",
                 "debounce_time_title": "Debounce Time (ms)",
 
+                "lp_hold_time_description": "If the click is longer than the specified time, we consider it long",
+                "lp_repeat_time_description": "The smaller, the faster the parameter changes",
+                "dp_time_description": "If there was no second click during the specified time, we consider the click a single one. 0 - disables all events except short and long",
```

Comment:
"If there is no second press during the specified time then press is single. 0 - disables all presses except short and long"